src/scripts/main.js

```diff
@@ -1,3 +1,76 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field tbody');
+const container = document.querySelector('.container');
+const [
+  appendRow,
```

Comment:
```suggestion
const appendRow = document.querySelector('.append-row');
```
It is better to specify each button